litestar/_asgi/routing_trie/mapping.py

```diff
@@ -182,38 +183,61 @@ def build_route_middleware_stack(
     Returns:
         An ASGIApp that is composed of a "stack" of middlewares.
     """
+
+    from litestar.contrib.opentelemetry import OpenTelemetryInstrumentationMiddleware
```

Comment:
I tried to base my hotreload plugin of this branch and I get:
```
    try:
        import opentelemetry  # noqa: F401
    except ImportError as e:
>       raise MissingDependencyException("opentelemetry") from e
E       litestar.exceptions.base_exceptions.MissingDependencyException: Package 'opentelemetry' is not installed but required. You can install it by running 'pip install litestar[opentelemetry]' to install litestar with the required extra or 'pip install opentelemetry' to install the package separately

```

so I start to think the private _patch_opentelemetry_middleware is designed to handle the case where opentelemetry isnt installed ?